store/storetest/compliance_store.go

```diff
@@ -104,7 +104,7 @@ func testComplianceExport(t *testing.T, ss store.Store) {
 
 	t1 := &model.Team{}
 	t1.DisplayName = "DisplayName"
-	t1.Name = "zz" + model.NewId() + "b"
+	t1.Name = NewTestId() + "b"
```

Comment:
Please remove all suffixes like `b`, `a` and others from here and everywhere else as well.

Some tests might fail if it tests for exactly those suffixes, but otherwise we should remove it.